SkateShop.Maui/Views/CartViewModel.cs

```diff
@@ -34,18 +31,19 @@ public partial class CartViewModel : ObservableValidator, IRecipient<CartItemAdd
         [IsTrue]
         private bool _hasAgreedToTermsAndConditions;
 
-        public CartViewModel()
+        public CartViewModel(IMessenger messenger)
         {
             _cartItems = new ObservableCollection<CartItemViewModel>();
 
-            StrongReferenceMessenger.Default.Register<CartItemAddedMessage>(this);
-            StrongReferenceMessenger.Default.Register<CartItemsRequestMessage>(this);
+            messenger.RegisterAll(this);
 
-            StrongReferenceMessenger.Default.Send<CartItemsRequestMessage>();
+            var response = messenger.Send<CartItemsRequestMessage>();
+            Receive(response);
         }
 
+
         [RelayCommand]
-        public async Task Checkout()
+        private async Task Checkout()
```

Comment:
Good catch! I assumed it would have to be `public` and didn't realize the generated command would be `public` anyways.